src/model/input.ts

```diff
@@ -8,129 +8,195 @@ const core = require('@actions/core');
-    return core.getInput('targetPlatform') || Platform.default;
+    return Input.getInput('targetPlatform') || Platform.default;
   }
 
   static get projectPath() {
-    const rawProjectPath = core.getInput('projectPath') || '.';
+    const rawProjectPath = Input.getInput('projectPath') || '.';
     return rawProjectPath.replace(/\/$/, '');
   }
 
   static get buildName() {
-    return core.getInput('buildName') || this.targetPlatform;
+    return Input.getInput('buildName') || this.targetPlatform;
   }
 
   static get buildsPath() {
-    return core.getInput('buildsPath') || 'build';
+    return Input.getInput('buildsPath') || 'build';
   }
 
   static get buildMethod() {
-    return core.getInput('buildMethod'); // processed in docker file
+    return Input.getInput('buildMethod') || ''; // processed in docker file
   }
 
   static get versioningStrategy() {
-    return core.getInput('versioning') || 'Semantic';
+    return Input.getInput('versioning') || 'Semantic';
   }
 
   static get specifiedVersion() {
-    return core.getInput('version') || '';
+    return Input.getInput('version') || '';
   }
 
   static get androidVersionCode() {
-    return core.getInput('androidVersionCode');
+    return Input.getInput('androidVersionCode');
   }
 
   static get androidAppBundle() {
-    const input = core.getInput('androidAppBundle') || false;
+    const input = Input.getInput('androidAppBundle') || false;
 
     return input === 'true';
   }
 
   static get androidKeystoreName() {
-    return core.getInput('androidKeystoreName') || '';
+    return Input.getInput('androidKeystoreName') || '';
   }
 
   static get androidKeystoreBase64() {
-    return core.getInput('androidKeystoreBase64') || '';
+    return Input.getInput('androidKeystoreBase64') || '';
   }
 
   static get androidKeystorePass() {
-    return core.getInput('androidKeystorePass') || '';
+    return Input.getInput('androidKeystorePass') || '';
   }
 
   static get androidKeyaliasName() {
-    return core.getInput('androidKeyaliasName') || '';
+    return Input.getInput('androidKeyaliasName') || '';
   }
 
   static get androidKeyaliasPass() {
-    return core.getInput('androidKeyaliasPass') || '';
+    return Input.getInput('androidKeyaliasPass') || '';
   }
 
   static get androidTargetSdkVersion() {
     return core.getInput('androidTargetSdkVersion') || '';
   }
 
   static get allowDirtyBuild() {
-    const input = core.getInput('allowDirtyBuild') || false;
+    const input = Input.getInput('allowDirtyBuild') || false;
 
     return input === 'true';
   }
 
   static get customParameters() {
-    return core.getInput('customParameters') || '';
+    return Input.getInput('customParameters') || '';
   }
 
   static get sshAgent() {
-    return core.getInput('sshAgent') || '';
+    return Input.getInput('sshAgent') || '';
   }
 
   static get gitPrivateToken() {
     return core.getInput('gitPrivateToken') || '';
+    // TODO  get from git auth status -t
```

Comment:
Leftover todo or intended?

Reply:
Will implement

Reply:
This comment refers to the behaviour of getting the current git auth token from the gh cli via gh auth status -t